website/src/docs/transloadit.md

```diff
@@ -412,3 +412,7 @@ uppy.on('transloadit:complete', (assembly) => {
 [assembly-status]: https://transloadit.com/docs/api/#assembly-status-response
 
 [template-credentials]: https://transloadit.com/docs/#how-to-create-template-credentials
+
+## Close behaviour
+
+When Uppy“s `close` method is called with `reason: 'user'`, the Transloadit plugin will cancel any Assemblies that were started as a result of uploads.
```

Comment:
```suggestion
## Assembly behavior when Uppy is closed

When integrating `@uppy/transloadit` with a UI plugin, such as `@uppy/dashboard`, closing Uppy will result in continuing assemblies on the server. This is controlled by Uppy's `close` and `cancelAll` methods, which receive `{ reason: 'unmount' }` by default. 

When the user manually cancels the upload any running assemblies will be cancelled (with `{ reason: 'user' }`.
```

Reply:
I'm not sure this is correct. I cannot see where in @uppy/dashboard it will call `close({ reason: 'unmount' })`. I can only see `reset()` in doneButtonHandler. But because we default to `{ reason: 'user' }` for backward compatibility, I think this will not be correct.

Reply:
Conceptually what I wrote it true but the technicalities where incorrect. I updated the comment

Reply:
I still think closing dashboard modal will result in assemblies being cancelled. But we can update the `reset` call inside dashboard/index.js to instead call `reset({ reason: 'unmount' })`, or what do you think?

Reply:
> I still think closing dashboard modal will result in assemblies being cancelled. But we can update the `reset` call inside dashboard/index.js to instead call `reset({ reason: 'unmount' })`, or what do you think?

I don't understand, vanilla dashboard won't cancel the assemblies, and we just made a change to do the same in frameworks. So what I wrote is correct?

Reply:
Ok, thanks for clarifying Artur. I didn't know that it would just hide it visually.

> When integrating `@uppy/transloadit` with `@uppy/dashboard`, closing the dashboard will result in continuing assemblies on the server. When the user manually cancels the upload any running assemblies will be cancelled.

So this behaviour is the same as before this PR, right?

Reply:
No, before this PR framework integrations of dashboard would cancel the assembly on close.